api/tests/test_api_v1.py

```
@@ -350,3 +350,167 @@ async def test_unauthenticated_requests_rejected(client):
+    bob_id = r.json()["id"]
+
+    # Create a group; duplicate names are rejected
+    r = await client.post("/api/v1/groups", json={"name": "prod", "description": "Production"})
+    assert r.status_code == 201, r.text
+    group_id = r.json()["id"]
+    r = await client.post("/api/v1/groups", json={"name": "prod"})
+    assert r.status_code == 400
+
+    # Add the host and bob to the group
+    r = await client.post(f"/api/v1/groups/{group_id}/hosts/{host_id}")
+    assert r.status_code == 200 and [h["id"] for h in r.json()["hosts"]] == [host_id]
```

Ruff (0.15.21)

[warning] Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)